doc/classes/Control.xml

```diff
@@ -1050,6 +1050,8 @@
 		<member name="rotation" type="float" setter="set_rotation" getter="get_rotation" default="0.0">
 			The node's rotation around its pivot, in radians. See [member pivot_offset] to change the pivot's position.
 		</member>
+		<member name="rotation_degrees" type="float" setter="set_rotation_degrees" getter="get_rotation_degrees">
+		</member>
```

Comment:
```suggestion
		<member name="rotation_degrees" type="float" setter="set_rotation_degrees" getter="get_rotation_degrees">
			Helper property to access [member rotation] in degrees instead of radians.
		</member>
```
Can be copy-pasted to others.